meta/documents/user_guide/en/initial_setup.md

```diff
@@ -0,0 +1,121 @@
+# Wayfair plugin: Initial Setup
+
+## Prerequisites
+* [Wayfair API credentials](obtaining_credentials.md).

```

Comment:
Would a supplier get to this page without previously setting up a relationship with us? If so we should put a prerequisite before this one that they must be an authorized supplier on Wayfair and point them to the process to kick it off.

Reply:
I had something in here originally but couldn't decide on wording. I'll re-visit this.